pylisa/src/main/java/it/unive/pylisa/symbolic/operators/ComparisonOperator.java

```diff
@@ -0,0 +1,105 @@
+package it.unive.pylisa.symbolic.operators;
```

Comment:
Is there a better location to place this minor class which is just a set of comparison operators? @lucaneg